index.js

```diff
@@ -34,13 +39,13 @@ module.exports = function (config) {
                 if (executedTest.tags[1].split("@")[1] == test.tags[1].split("@")[1]) {
                     //we add the first test result status in exemples array
                     if (executedTest.examples.length == 0) {
-                        if (executedTest.state == "passed") executedTest.examples.push("PASS");
-                        else executedTest.examples.push("FAIL");
+                        if (executedTest.state == "passed") executedTest.examples.push("PASSED");
```

Comment:
you should leaved it to pass. unless it's dont work on cloud.
in that case, you can asomething like
`if (xray_cloud) executedTest.examples.push("PASSED");
else executedTest.examples.push("PASS");`